client/src/store/utils/reducerFunctions.js

```diff
@@ -1,7 +1,7 @@
 export const addMessageToStore = (state, payload) => {
-  const { message, sender } = payload;
-  // if sender isn't null, that means the message needs to be put in a brand new convo
-  if (sender !== null) {
+  const { message, sender, exists } = payload;
+  // if not exists, then make a new convo
+  if (exists !== true && exists !== 1) {
```

Comment:
Is there a particular case where `exists` would be 1?

Reply:
I am not sure why, but the exists property gets translated into a 1 from true. I printed from both sides (sending and receiving), the sender's exists is true and the receiver is 1. I've just decided to keep everything 1 or 0 instead of true or false now.

Reply:
It turns out the default socket.io parser apparently doesn't support booleans. 1 and 0 are truthy and falsy, respectively, so they should still work in any conditions you provide them to. Here, specifically, you could just use `!exists` which will treat it as a boolean.